Lib/_ast_unparse.py

```diff
@@ -0,0 +1,1150 @@
+# This module contains ``ast.unparse()``, defined here
+# to improve the import time for the ``ast`` module.
+import sys
+from _ast import *
+from ast import NodeVisitor
```

Comment:
This cyclic dependency seems unfortunate.

Reply:
It is technically cyclic, but there are no issues in practice: the module-level `__getattr__` is defined after the `NodeVisitor` class. `from ast import unparse` and `ast.unparse` both still work.